adwaita-icon-theme.yaml

```diff
@@ -0,0 +1,8 @@
+name: adwaita-icon-theme
```

Comment:
It's already installed with Gtk3.

Reply:
Oh you seem to be using gtk2 instead of gtk3. Is there a reason why you don't use gtk3 instead?

Reply:
> Oh you seem to be using gtk2 instead of gtk3. Is there a reason why you don't use gtk3 instead?

- The stable release of Gpick doesn't support Gtk+ 3 at all, and [Flathub only hosts stable application releases](https://github.com/flathub/flathub/wiki/App-Requirements#stable-releases-reproducible-builds).
- Development version of Gpick has support for both Gtk+2 and Gtk+ 3. However, it has several regressions compared to the stable release. For example, [resolving color names](https://raw.githubusercontent.com/scx/gpick-flatpak/master/gpick-flatpak.png) (one of the main features) doesn't work at all.
- Gtk+3 version of Gpick is totally [useless](https://github.com/thezbyg/gpick/issues/133) under Wayland session. I also experienced some problems under X11.
- Many distributions (e.g. Debian and Ubuntu) provide a stable release (Gtk+2 version) of Gpick. Fedora provides a development version (Gtk+3 version), and this package is very unstable.
- There could be some minor problems with the layout in Gtk+3 version. In my opinion the interface is a bit illegible due to the lack of paddings and margins, especially combined with flat controls.

Reply:
@nedrichards 

Yeah, that's the problem. Anyway, in my opinion, the development version does not provide any benefits, but has a several regressions.

Reply:
@nedrichards 

However, it should be possible to do that somehow. For example, the [Peek](https://github.com/phw/peek) is able to record desktop even under Wayland session.

Reply:
Recording the desktop uses the screen cast API. So yes, it could call that. https://wiki.gnome.org/Projects/Mutter/RemoteDesktop

Overall, gtk2 apps are less good within Flatpak, there isn't a portal implementation and themes aren't supported for instance - and that's quite aside from the fact that GTK2 doesn't support modern features such as HIDPI. Sad news if it's not up to scratch.

Reply:
@nedrichards 

As I mentioned earlier, Gtk+3 support is available only in development version. The stable release (0.2.5) doesn't support Gtk+3 at all, and [Flathub only hosts stable application releases](https://github.com/flathub/flathub/wiki/App-Requirements#stable-releases-reproducible-builds).

Reply:
We bend this rule when it makes sense, However I tested GTK3-based gpick from distro repositories locally and it seems useless either way on Wayland so I don't see a reason to use RC release.